docs/book/events.md

```diff
@@ -51,7 +51,7 @@ final class ConsoleEventDispatcherFactory
     public function __invoke(ContainerInterface $container): EventDispatcher
     {
         $dispatcher = new EventDispatcher();
-        $dispatcher->addListener($container->get(ErrorListener::class));
+        $dispatcher->addSubscriber($container->get(ErrorListener::class));
```

Comment:
`addListener` is still referenced  elsewhere in the document

Reply:
@Ocramius addListener is still a valid way to add a listener if it's callable. It's just won't work for ErrorListener as it implements EventSubscriberInterface instead.

Reply:
Ah, on second reading, I see your point.

Reply:
@dkmuir 
Unfortunately you have misunderstood something here. There is another reference in the text to the `addListener` method:

https://github.com/laminas/laminas-cli/blob/d4b0665f8f2be1517973e192f4d6b175c66b571e/docs/book/events.md?plain=1#L8

Reply:
Comment timing conflict :D 

Is now resolved